pyface/dock/dock_sizer.py

```diff
@@ -366,12 +366,18 @@ class DockItem(HasPrivateTraits):
     # The parent of this item:
     parent = Any()
 
+    # The control of this item:
+    control = Any()
+
     # The DockWindow that owns this item:
     owner = Property(observe="parent")
 
     # Bounds of the item:
     bounds = Bounds
 
+    # The user-visible name of the item:
+    name = Str()
+
```

Comment:
notice the commented out code below on lines 429-433.  

Not sure why that code ever got commented out...
It also has control as an Instance( wx.control)

Reply:
ahh. nice catch. i'll do some digging and make changes before merging. i had very strong blinders on when i was working on this change.

Reply:
Looks like this goes back all the way to the second commit in the git repo (the first commit seems empty) - https://github.com/enthought/pyface/commit/e569f0953f0b4c3a0ff0a8ba8767ef9bcf80f81a - so i'll actually remove the commented code while updating the trait definitions.

I tested this with enable kiva_explorer and it still works.